app/templates/components/slider-control.hbs

```diff
@@ -0,0 +1,5 @@
+<div>
+  <span class="active-segment" style="width: {{percentage}}%"></span>
```

Comment:
Can't we use input with type slider? Would be much easier to maintain the code without all this calculation of percentage